lib/core/nav/HeaderNav.js

```diff
@@ -172,8 +179,16 @@ class HeaderNav extends React.Component {
       // set link to blog url
       href = this.props.baseUrl + 'blog';
     }
+    const itemClasses = classNames({
+      siteNavGroupActive:
+        (link.doc && docGroupActive) || (link.blog && this.props.current.blog),
+      siteNavItemActive:
+        docItemActive ||
+        (link.blog && this.props.current.blogListing) ||
```

Comment:
I'm not sure what's going on with the blog but it doesn't seem to be highlighted when I'm looking at the blog section. All the other sections are working fine (thanks for fixing this!!)

Reply:
Nevermind, I actually needed to change the headerLink from

```js
    { href: "/blog", label: "Blog" },
```

to

```js
    { blog: true, label: "Blog" },
```

now it works :)

Reply:
Is this a backward-incompatible change? Do we need to document this somewhere or support the previous?

Reply:
We added the corresponding documentation when adding this feature to explain it doesn't work with "href" header links: https://docusaurus.io/docs/en/navigation.html#active-links-in-site-navigation-bar

Reply:
It's not really backward incompatible, href: "/blog" still works but doesn't have the highlight (note that it didn't before either!). If you are saying to use blog: true in the docs, this should be fine, I wrote this before this boolean existed.